src/canvas.rs

```diff
@@ -338,12 +311,6 @@ impl Painter {
                 for battery_widget in app_state.battery_state.widget_states.values_mut() {
                     battery_widget.tab_click_locs = None;
                 }
-
-                // Reset column headers for sorting in process widget...
-                for proc_widget in app_state.proc_state.widget_states.values_mut() {
-                    proc_widget.columns.column_header_y_loc = None;
-                    proc_widget.columns.column_header_x_locs = None;
-                }
```

Comment:
No longer needed since we've moved into just handling this step in table rendering step.